src/Model/Tag.php

```diff
@@ -386,35 +390,36 @@ public static function remove(int $uriid, int $type, string $name, string $url =
 			return;
 		}
 
-		Logger::info('Removing tag/mention', ['uri-id' => $uriid, 'tid' => $tag['tid'], 'name' => $name, 'url' => $url, 'callstack' => System::callstack(8)]);
-		DBA::delete('post-tag', ['uri-id' => $uriid, 'type' => $type, 'tid' => $tag['tid'], 'cid' => $tag['cid']]);
+		Logger::info('Removing tag/mention', ['uri-id' => $uriId, 'tid' => $tag['tid'], 'name' => $name, 'url' => $url, 'callstack' => System::callstack(8)]);
+		DBA::delete('post-tag', ['uri-id' => $uriId, 'type' => $type, 'tid' => $tag['tid'], 'cid' => $tag['cid']]);
 	}
 
 	/**
 	 * Remove tag/mention
 	 *
-	 * @param integer $uriid
-	 * @param string $hash
-	 * @param string $name
-	 * @param string $url
+	 * @param integer $uriId URI id
+	 * @param string $hash Hash
+	 * @param string $name Name
+	 * @param string $url URL
+	 * @return void
 	 */
-	public static function removeByHash(int $uriid, string $hash, string $name, string $url = '')
+	public static function removeByHash(int $uriId, string $hash, string $name, string $url = '')
 	{
 		$type = self::getTypeForHash($hash);
 		if ($type == self::UNKNOWN) {
 			return;
 		}
 
-		self::remove($uriid, $type, $name, $url);
+		self::remove($uriId, $type, $name, $url);
 	}
 
 	/**
 	 * Get the type for the given hash
 	 *
-	 * @param string $hash
-	 * @return integer type
+	 * @param string $hash Hash
```

Comment:
This wasn't necessary.
```suggestion
	 * @param string $hash
```